pkg/goods/support/host-support-bundle.tmpl.yaml

```diff
@@ -448,3 +606,38 @@ spec:
         - pass:
             when: "false"
             message: "No signs of hostname changes found"
+  - textAnalyze:
+      checkName: Check if localhost resolves to 127.0.0.1
+      fileName: host-collectors/run-host/localhost-ips.txt
+      regex: 'localhost has address 127.0.0.1'
+      outcomes:
+        - fail:
+            when: "false"
+            message: "'localhost' does not resolve to 127.0.0.1 ip address"
```

Comment:
I think we need to make a pass to make these match the new EC style of more informative action oriented messages.

Something like: localhost doesn't resovle to 127.0.0.1 add <hostname> to /etc/hosts to resolve.

Reply:
done